src/components/ProfilePage/index.jsx

```diff
@@ -158,32 +166,29 @@ class ProfilePage extends Component {
           </button>
         </div>
         <div className="userInfo">
+          <h1 className="profileName">
+            {firstName} {lastName}
+          </h1>
+          {profileId !== userId && !(userId in friends) && (
+            <button
+              type="button"
+              className="addFriendButton"
+              onClick={this.addFriend}
+            >
+              + Add Friend

```

Comment:
make all caps

Reply:
also make the cursor a pointer on. in css just select the button (not hover) and `cursor: pointer;`